polygon/bridge/db.go

```diff
@@ -165,3 +166,84 @@ func GetEvents(ctx context.Context, db *polygoncommon.Database, id IDRange) ([][
 
 	return events, err
 }
+
+// Map operations
+
+func StoreMap(ctx context.Context, db *polygoncommon.Database, eventMap map[uint64]IDRange) error {
```

Comment:
`StoreMap`, `GetMap`, `UnwindMap` - the usage of `Map` in the naming is too abstract. It is not clear what you mean by `Map`. Also in future there may be other `Maps` for other mappings. It would be good to be a bit more specific with naming. In this case the mapping is `BlockNumToEventRange` (however note I think we may want it to be `BlockNumToStartEvent` as per my previous comment - https://github.com/ledgerwatch/erigon/pull/10615/files#r1627629086)